packages/sirius-web/frontend/sirius-web-application/src/modals/upload-document/UploadDocumentModal.types.ts

```diff
@@ -17,4 +17,17 @@ export interface UploadDocumentModalProps {
 
 export interface UploadDocumentModalState {
   file: File | null;
+  readOnly: boolean;
+}
+
+export interface UploadDocumentSplitButtonProps {
+  readOnly: boolean;
+  onPermissionSelected: (readOnly: boolean) => void;
```

Comment:
I will update this to remove any implication of "permission", we don't provide support for permissions